webroot/src/components/PrivilegeEventNode/PrivilegeEventNode.vue

```diff
@@ -0,0 +1,34 @@
+<!--
+    PrivilegeEventNode.vue
+    CompactConnect
+
+    Created by InspiringApps on 3/21/2025.
+-->
+
+<template>
+    <div
+        class="event-node-container"
+        role="listitem"
+        :aria-label="$t('licensing.eventNodeLabel', { eventNameDisplay, eventDate })"
+        :class="{
+            'short': eventLengthBucket === 'short',
+            'medium': eventLengthBucket === 'medium',
+            'long': eventLengthBucket === 'long'
+        }"
```

Comment:
I assume the separation between CSS class and variable value is intentional? Given this could also just be 
```suggestion
        :class="eventLengthBucket"
```

Reply:
actually not intentional, good call